src/components/mediaControls.js

```diff
@@ -130,6 +127,30 @@ class MiniMediaPlayerMediaControls extends LitElement {
     }
   }
 
+  renderPlayButtons(hide) {
+    if (!hide.play_pause) {
+      return html`
+                <paper-icon-button
+                  @click=${e => this.player.playPause(e)}
+                  .icon=${ICON.PLAY[this.player.isPlaying]}>
+                </paper-icon-button>
+                ${!hide.play_stop ? html`
+                <paper-icon-button
+                  @click=${e => this.player.stop(e)}
+                  .icon=${ICON.STOP}>
+                </paper-icon-button>
+                ` : html``}
+                `;
+    } else if (!hide.play_stop) {
+      return html`
+                <paper-icon-button
+                  @click=${e => this.player.playStop(e)}
+                  .icon=${this.player.isPlaying ? ICON.STOP : ICON.PLAY[this.player.isPlaying]}>
```

Comment:
`.icon=${this.player.isPlaying ? ICON.STOP : ICON.PLAY[this.player.isPlaying]}>`

Last part could be just `ICON.PLAY[false]` since we already check for `isPlaying` in the condition expression and we will never reach the else condition if `isPlaying` equals `true`.